app/components/common/Feedback.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import { showReportDialog } from '@sentry/browser';
+import { captureException, lastEventId } from '@sentry/react';
+import { v4 as uuidv4 } from 'uuid';
+import { smColors } from '../../vars';
+
+const Container = styled.div`
+  padding: 4px 12px;
+  bottom: 10px;
+  display: flex;
+`;
+
+const Button = styled.div`
+  font-size: 12px;
+  text-decoration: underline;
+  border: 0;
+  color: ${({ theme }) =>
+    theme.isDarkMode ? smColors.lightGray : smColors.darkGray};
+  cursor: pointer;
+`;
+
+const FeedbackButton = () => {
+  return (
+    <Container>
+      <Button
+        onClick={() => {
+          captureException(`client_report_${uuidv4()}`);
```

Comment:
It's strange, but it puts all the stuff under the same event 🤔 
Mb it will be more robust to make it like this:
```suggestion
          const eventId = captureException(`client_report_${uuidv4()}`);
          showReportDialog({ eventId });
```

Reply:
Oh, just found that actually it sends reports with different `eventId`s, but Sentry groups it somehow 🤔 
Also I'd prefer to have a clear event name, like `BugReport` instead of `EN`.